rundown/resources/schedule.py

```diff
@@ -0,0 +1,35 @@
+from pydantic import BaseModel, validator
+
+from rundown.resources.validators import change_timezone
+
+
+class BaseSchedule(BaseModel):
```

Comment:
what's the purpose of breaking it into a `BaseSchedule` and `Schedule` class?

Reply:
I noticed that `GET /sports/<sport-id>/events/<date>` returns a subset of schedule data compared to `GET /sports/<sport-id>/schedule`, so I separated it into `BaseSchedule` and `Schedule`.

Event route example return: 
```
"schedule": {
  "season_type": "Regular Season",
  "season_year": 2021,
  "event_name": "Detroit at Tampa Bay - 2021-04-03",
  "attendance": "3800"
},
```

Schedule route example return:
```
{
  "id": "91570",
  "event_uuid": "11eb-967b-cb3f5000-80e8-e33c8bac5ab1",
  "event_id": "cdccf7984487550a49d3ba746f651603",
  "sport_id": 6,
  "season_type": "Regular Season",
  "season_year": 2021,
  "away_team_id": 121,
  "home_team_id": 120,
  "away_team": "Arizona",
  "home_team": "Los Angeles",
  "date_event": "2021-04-06T02:00:00Z",
  "neutral_site": false,
  "conference_competition": false,
  "away_score": 0,
  "home_score": 0,
  "league_name": "National Hockey League",
  "event_name": "Arizona at Los Angeles - 2021-04-06",
  "event_location": "STAPLES Center",
  "attendance": 0,
  "updated_at": "2021-04-05T06:11:05.328635Z",
  "event_status": "STATUS_SCHEDULED",
  "event_status_detail": "4/5 - 10:00 PM EDT"
}
```